im-gespraech-buber-gesellschaft.csl

```diff
@@ -0,0 +1,428 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only">
+  <info>
+    <title>Im Gespräch - Hefte der Martin Buber-Gesellschaft (German)</title>
+    <title-short>Im Gespräch</title-short>
+    <id>http://www.zotero.org/styles/im-gespraech-buber-gesellschaft</id>
+    <link href="http://www.zotero.org/styles/im-gespraech-buber-gesellschaft" rel="self"/>
```

Comment:
below is great. You can simply link to your comment as documentation, i.e.
add ```<link href="https://github.com/citation-style-language/styles/pull/1674/files#r38487900" rel="documentation"/>``` right here. If you'd rather, you can also upload this in any other format anywhere else and link to it as long as it's going to be reasonably permanent. The point of this is that we want to have a reference if someone reports an error.